src/components/RefineSearch/RefineSearch.tsx

```diff
@@ -78,10 +77,11 @@ const RefineSearch = ({
         )
       } else return null
     })
-    .concat(dateFieldset)
+    .concat(<DateForm {...dateFormProps} />)
 
   const handleSubmit = (e: SyntheticEvent) => {
     e.preventDefault()
+    if (validateDateRange() === false) return
```

Comment:
nitpick: you can do `!validateDateRange()` similar to how its set in `/pages/search/advanced.tsx`.

Reply:
updated